### PR TITLE
[pt-br] Include dockershim-faq translation

### PR DESCRIPTION
**Pull request by marcelomansur**
This pull request aims to include pt-br translation for [dockershim-faq](https://kubernetes.io/blog/2022/02/17/dockershim-faq/).

/assign @edsoncelio

### CODE REVIEW

File: content/pt-br/blog/_posts/2022-02-17-updated-dockershim-faq.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+terá um desempenho estritamente melhor e menos sobrecarga. No entanto, nós encorajamos você a
+explorar todas as opções do [cenário CNCF], pois outro agente de execução de contêiner
+pode funcionar ainda melhor para o seu ambiente.
+
+[cenário CNCF]: https://landscape.cncf.io/card-mode?category=container-runtime&grouping=category
+
+### O que devo ficar atento ao mudar a minha implementação de CRI utilizada?
+
+Embora o código de conteinerização base seja o mesmo entre o Docker e a maioria
+CRIs (incluindo containerd), existem algumas poucas diferenças. Alguns
```
Comment (edsoncelio) [inline content/pt-br/blog/_posts/2022-02-17-updated-dockershim-faq.md:152]:
```suggestion
dos CRIs (incluindo containerd), existem algumas poucas diferenças. Alguns
```

**Review by edsoncelio — CHANGES_REQUESTED** [review]
Thanks @marcelomansur, I left a comment about just a small typo, after that it's an LGTM for sure!

File: content/pt-br/blog/_posts/2022-02-17-updated-dockershim-faq.md (hunk 1 above)
Comment (marcelomansur) [reply content/pt-br/blog/_posts/2022-02-17-updated-dockershim-faq.md:152]: Thanks @edsoncelio! I've just submitted a new commit including this correction :smile:

Comment (marcelomansur) [reply content/pt-br/blog/_posts/2022-02-17-updated-dockershim-faq.md:152]: Thanks @edsoncelio! I've just submitted a new commit including this correction :smile:

**Review by edsoncelio — APPROVED** [review]
Thanks!

/lgtm

**Pull request merged**